### DISCUSSION TITLE
Messages encapsulation in a .resx file

### DISCUSSION THREAD

**Discussion by N1K0232** [marcominerva/SearchGPT#3] [Q&A]
I noticed that in the ChatService file, there's a message saved in a const string field.
It's better and useful to encapsulate it in a .resx file isn't it?

Would it also be useful for implementing localization with multiple languages?

**Comment by marcominerva** [comment]
This project is a Proof-Of-Concept. In a real world scenario, your suggestion will surely be correct.
